dotrix_core/src/assets.rs

```diff
@@ -90,7 +90,26 @@ impl Assets {
 
     /// Imports an asset file by its path and returns [`Id`] of the [`Resource`]
     pub fn import(&mut self, path_str: &str) -> Id<Resource> {
-        let path = std::path::Path::new(path_str);
+        // TODO: implement conditional import logic by device type
+        let path_buf = match std::path::Path::new(path_str).exists() {
+            true => {
+                let mut new_path = std::path::PathBuf::new();
+                new_path.push(path_str);
+                new_path
+            }
+            false => {
+                let mut new_path = std::env::current_exe()
+                    .map(|path| {
+                        path.parent()
+                            .map(|exe_parent_path| exe_parent_path.to_owned())
+                            .unwrap()
+                    })
```

Comment:
Do I understand it right, that for iOS we need an absolute path always?

Reply:
If you access in relative path, it always start with root path which is `/`. So if we access with `assets/models/image.png`, it is `/assets/models/image.png`. I checked in my case. if you can double check it would be great!

Reply:
OK, I will check it. 
In any case, I like the idea to have absolute paths for assets.
I would add `root` property to `Assets` and default it to the root of executable. 
For each asset we will construct the final path from the root and user defined path string.
A developer will be able to change the `root` for the assets at `startup`.

What do you think?

Reply:
LGTM

Reply:
Give a chance to this one: https://github.com/lowenware/dotrix/pull/146/files